src/modules/75-cf/components/TableFilters/TableFilters.tsx

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2022 Harness Inc. All rights reserved.
+ * Use of this source code is governed by the PolyForm Shield 1.0.0 license
+ * that can be found in the licenses directory at the root of this repository, also available at
+ * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
+ */
+
+import React from 'react'
+import { Layout } from '@harness/uicore'
+import type { StringKeys } from 'framework/strings'
+import { FilterCard } from './FilterCard'
+
+interface TableFiltersProps {
+  filters: Array<{
+    labelStringKey: StringKeys
+    total: number
+    queryProps: { key: string; value: string | boolean | number }
```

Comment:
This looks kinda complex, could you just let the called pass an object?
```suggestion
    queryProps: Record<string, unknown>
```

So when then in your `FlagTableFilters` you could then do something like.
```
queryProps: { [FlagFilterKeys.ENABLED]: true }
```

Reply:
As with seemingly everything in this job _it depends_. Explicit is ideal a lot of times. Maybe instead of `unknown`, you could use `string | boolean | number` as a compound type. I miss the higher level `SCALAR` type from PHP that covers strings, booleans, numbers etc. `unknown` and `any` are kinda get-out-of-jail-free cards when you want to allow JS's natural loose typing to take over (well with `unknown`, if you use it you'll then need to assert its actual type, but we're handing off to a 3rd party here who accepts `any`, so it should be fine).